titus-server-master/src/main/java/com/netflix/titus/master/kubernetes/controller/PersistentVolumeUnassociatedGcController.java

```diff
@@ -120,6 +134,38 @@ public boolean gcItem(V1PersistentVolume pv) {
         return false;
```

Comment:
return `true` when it is not found in the `catch(ApiException)` block?

Reply:
Yes, good catch!